src/app/session.service.ts

```diff
@@ -1,15 +1,35 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { Http } from '@angular/http';
+import { GlobalSettingsService } from './globalSettings.service';
 
 @Injectable()
 export class SessionService {
+  config;
```

Comment:
space between constructor and property.  also I think a space after the class declaration and property.